platform/android/export/export.cpp

```diff
@@ -824,17 +836,32 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 							encode_uint32(min_gles3 ? 0x00030000 : 0x00020000, &p_manifest.write[iofs + 16]);
 						}
 
+						if (tname == "uses-feature" && attrname == "required" && string_table[attr_value] == "require_hardware_camera_arcore") {
+							string_table.write[attr_value] = xr_mode_index == XR_MODE_ARCORE ? "true" : "false";
+						}
+
 						if (tname == "meta-data" && attrname == "name" && string_table[attr_value] == "xr_mode_metadata_name") {
 							// Update the meta-data 'android:name' attribute based on the selected XR mode.
-							if (xr_mode_index == 1 /* XRMode.OVR */) {
+							if (xr_mode_index == XR_MODE_OVR) {
 								string_table.write[attr_value] = "com.samsung.android.vr.application.mode";
+							} else if (xr_mode_index == XR_MODE_ARCORE) {
+								string_table.write[attr_value] = "com.google.ar.core";
 							}
 						}
 
 						if (tname == "meta-data" && attrname == "value" && string_table[attr_value] == "xr_mode_metadata_value") {
```

Comment:
Somehow this line, which isn't showing a change, is throwing the same error on Export as the one below on line 864, that's extra confusing to me, and definitely makes me think I'm doing something wrong

Reply:
Fixed.